src/audio/google_rtc_audio_processing.c

```diff
@@ -347,16 +347,16 @@ static int google_rtc_audio_processing_copy(struct comp_dev *dev)
 	struct google_rtc_audio_processing_comp_data *cd = comp_get_drvdata(dev);
```

Comment:
I forgot to mention in commit text that the MCPS saving was tested with 48 kHz stereo 16 bit capture and reference streams in a TGL-H HDA generic test device.